### DISCUSSION TITLE
Configure SecurityContext for CSI data movement helper Pods

### DISCUSSION THREAD

**Discussion by Elias-elastisys** [vmware-tanzu/velero#7537] [Community support Q&A] [answered]
Hi.

I've been trying out the CSI data movement feature in Velero and am having issues with the helper Pods that velero creates when I try to backup PV/PVCs without any Pods using the resource. I work in a quite restricted environment that requires numeric users to verify that they are not root, and I was wondering if there is any way of configuring the helper Pods that velero creates to not use the default `cnb` user?

For restores we have previously used [this configmap](https://velero.io/docs/v1.13/file-system-backup/#customize-restore-helper-container) as described in your docs. But I can't find any documentation/issues/questions/code for a similar thing for the Pods that are created from the DataUpload resource when using this new csi data movement. Is it even possible?

**Comment by Lyndon-Li** [comment]
Do you want to run the backupPod and the data mover under a non-root user?

**Reply by Elias-elastisys** [reply]
Yes I need to run them as a numeric non-root user (e.g. 1000 or 10000)

**Comment by Lyndon-Li** [comment, marked as answer]
This is not currently supported. Please wait for resolve of this issue #7198

**Reply by Elias-elastisys** [reply]
Alright thanks! I solved it in the meantime with Gatekepeer Mutations.
